Detectors/ITSMFT/ITS/simulation/src/Detector.cxx

```diff
@@ -326,14 +295,11 @@ Bool_t Detector::ProcessHits(FairVolume* vol)
   if (notSens)
     return kFALSE; // RS: can this happen? This method must be called for sensors only?
 
+  // Is it needed to keep a track reference when the outer ITS volume is encountered?
   auto stack = (o2::Data::Stack*)fMC->GetStack();
-  if (fMC->IsTrackExiting() && (lay == 0 || lay == 6)) {
-    // Keep the track refs for the innermost and outermost layers only
-    o2::TrackReference tr(*fMC, GetDetId());
-    tr.setTrackID(stack->GetCurrentTrackNumber());
-    tr.setUserId(lay);
-    stack->addTrackReference(tr);
-  }
```

Comment:
@mario6829 Please, restore these lines. They are needed for doing the comparison with MC at the outermost layer.